src/pages/Login/Login.scss

```
@@ -0,0 +1,65 @@
+.top {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 300px;
+  margin-right: 300px;
+  .topback {
+    border: none;
+    color: black;
+    font-size: 20px;
+  }
```

```suggestion
.top {
  display: flex;
  justify-content: space-between;
  margin-left: 300px;
  margin-right: 300px;
  
  .topback {
    border: none;
    color: black;
    font-size: 20px;
  }
```
css 작성컨벤션 지켜주세요! className선택자와 상단 선택자의 마지막 style코드 사이에는 공백이 필요합니다.
해당내용은 모든 곳을 다 찾으셔서 수정해주세요

---

네